mapper/mapper.go

```diff
@@ -268,7 +268,13 @@ func CreateVersionsList(ctx context.Context, d dataset.Model, edition dataset.Ed
 			})
 		}
 
-		version.Reason = "-" // TODO: use reason from dataset api if it becomes available
+		var correctionReasons []string
+		for _, alert := range *ver.Alerts {
+			if alert.Type == "correction" {
```

Comment:
Is it worth making this a const or defining it before the loop? Efficiency n all that...